app/helpers/metadata_helper.rb

```diff
@@ -19,15 +19,39 @@ def metadata_url
 
   def format_title(title)
     return "Exercism" unless title
-
     "#{title} | Exercism"
   end
 
   def metadata
     @metadata ||=
       case namespace_name
+      when "admin"
+        { title: "Admin" }
       when "my"
-        nil
+        case controller_name
+        when "notifications"
+          { title: "Notifications" }
+        when "settings", "track_settings", "preferences"
+          { title: "Settings" }
+        when "reactions"
+          { title: "My Reactions" }
+        when "solutions"
+          { title: "My #{@track.title}/#{@exercise.title}" }
+        when "tracks"
+          case action_name
+          when "index"
+            { title: "My Tracks" }
```

Comment:
Can you change this to be `"Track"` pls?

Reply:
This page is a list of all the tracks and which ones the user belongs to, so I am not sure if the singular is correct.

Reply:
Sorry. You're very right. I meant `"Tracks"`